### PR TITLE
Cobbler IPMI configuration and utilities

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

File: roles/cobbler/templates/utils/reimage.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+## {{ ansible_managed }}
+set -ex
+name=$1
+profile=$2
+echo "Reimaging $name with profile $profile"
+cobbler system edit --name=$name netboot off
```
Comment (dmick) [inline roles/cobbler/templates/utils/reimage.sh]: why set to off first?

File: roles/cobbler/tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ---
+- include: ipmi_secrets.yml
```
Comment (andrewschoen) [inline roles/cobbler/tasks/main.yml]: I might move the ``always`` tag here instead of having two of them in ipmi_secrets.yml

Comment (zmc) [inline roles/cobbler/tasks/main.yml]: True; I had put them there initially in case it needed to be included from a second place. I can move them though.

**Comment by zmc** [comment]
Anything else @andrewschoen ?

**Comment by andrewschoen** [comment]
@zmc nope, just didn't see the update. :)

**Pull request merged**